js/content/community.js

```diff
@@ -2815,6 +2816,11 @@ let MarketListingPageClass = (function(){
         }
     };
 
+    MarketListingPageClass.prototype.addPriceHistoryZoomControl = function() {
+        // DOMPurify would delete the onclick property
```

Comment:
Can we use DOMPurify and adding click event via addEventListener?

Reply:
Sure